shared/src/commonMain/kotlin/org/hyperskill/app/notification_onboarding/domain/analytics/NotificationsOnboardingPermissionResultHyperskillAnalyticsEvent.kt

```diff
@@ -0,0 +1,30 @@
+package org.hyperskill.app.notification_onboarding.domain.analytics
+
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticAction
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticEvent
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticPart
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticRoute
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticTarget
+
+/**
+ * Represents allowing or denying of the system notification permission.
+ *
+ * JSON payload:
+ * ```
+ * {
+ *     "route": "/onboarding/notifications",
+ *     "action": "click",
+ *     "part": "notifications_system_notice",
+ *     "target": "allow"
+ * }
+ * ```
+ * @see HyperskillAnalyticEvent
+ */
+class NotificationsOnboardingPermissionResultHyperskillAnalyticsEvent(
```

Comment:
We already have analytic events for the system prompt dialog:

1. https://github.com/hyperskill/mobile-app/blob/develop/shared/src/commonMain/kotlin/org/hyperskill/app/notification/local/domain/analytic/NotificationSystemNoticeShownHyperskillAnalyticEvent.kt
2. https://github.com/hyperskill/mobile-app/blob/develop/shared/src/commonMain/kotlin/org/hyperskill/app/notification/local/domain/analytic/NotificationSystemNoticeHiddenHyperskillAnalyticEvent.kt